docs/hacking/building-from-source.md

```diff
@@ -0,0 +1,27 @@
+# Building from Source
+
+## Prerequisites
+
+Before you begin, ensure you have the following software installed on your system:
+
+- Git
+- Yarn
+- Node.js version 22 or higher
+- Soufflé (Refer to the [official installation guide](https://souffle-lang.github.io/install) for instructions)
+
+## Cloning the Repository
+
+Clone the Misti repository:
+
+```bash
+git clone 'https://github.com/nowarp/misti'
+```
+
+## Building the Project
```

Comment:
Could we also add a section regarding running the analyzer during the development? I mean, `yarn misti`. Maybe, we should include an example on how to run it for tests: `yarn misti test/good/never-accessed.tact`.

Reply:
Maybe we should also refer to HACKING.md which contains some information about playing with Misti internals: https://github.com/nowarp/misti/blob/master/HACKING.md